java/dataflow-connector-examples/src/main/java/com/google/cloud/bigtable/dataflow/example/HelloWorldWrite.java

```diff
@@ -71,44 +74,51 @@ public void processElement(DoFn<String, Mutation>.ProcessContext c) throws Excep
       c.output(new Put(c.element().getBytes()).addColumn(FAMILY, QUALIFIER, VALUE));
     }
   };
+  // [END bigtable_dataflow_connector_process_element]
 
   /**
    * <p>Creates a dataflow pipeline that creates the following chain:</p>
    * <ol>
-   *   <li> Puts an array of "Hello", "World" into the Pipeline
-   *   <li> Creates Puts from each of the words in the array
-   *   <li> Performs a Bigtable Put on the items in the
+   * <li> Puts an array of "Hello", "World" into the Pipeline
+   * <li> Creates Puts from each of the words in the array
+   * <li> Performs a Bigtable Put on the items in the
    * </ol>
    *
    * @param args Arguments to use to configure the Dataflow Pipeline.  The first three are required
-   *   when running via managed resource in Google Cloud Platform.  Those options should be omitted
-   *   for LOCAL runs.  The last four arguments are to configure the Bigtable connection.
-   *        --runner=BlockingDataflowPipelineRunner
-   *        --project=[dataflow project] \\
-   *        --stagingLocation=gs://[your google storage bucket] \\
-   *        --bigtableProject=[bigtable project] \\
-   *        --bigtableInstanceId=[bigtable instance id] \\
-   *        --bigtableTableId=[bigtable tableName]
+   * when running via managed resource in Google Cloud Platform.  Those options should be omitted
+   * for LOCAL runs.  The last four arguments are to configure the Bigtable connection.
+   * --runner=BlockingDataflowPipelineRunner --project=[dataflow project] \\
```

Comment:
Should this be enclosed in `<code>` or `<verbatim>`?

Reply:
Done